sound/soc/sof/Kconfig

```diff
@@ -128,6 +128,14 @@ config SND_SOC_SOF_DEBUG_ENABLE_DEBUGFS_CACHE
 	  Say Y if you want to enable caching the memory windows.
 	  If unsure, select "N".
 
+config SND_SOC_SOF_DISABLE_DMA_TRACE
+	bool "SOF disable DMA-based trace"
+	help
+	  This option disables the firmware trace completely. This
+	  option is not recommended and should only be used to
+	  work-around DMA problems.
```

Comment:
I have question for a long time: Do we need to enable DMA TRACE in production release ?
If DMA TRACE should be disabled, the description is not fit for this case

Reply:
Only reason I see is a practical one. Given DMA trace is using some of the same resources as production data flows, one can trigger different bugs with it and without. So if we disable in production (which seems sensible), the developer base should start by default validating their changes without the trace on, and only enable trace when an issue happens. But this is rather common state of affairs, so disabling by default would seem the sensible approach.

Reply:
it's should be like dynamic debug (dev_dbg), either statically removed or enabled by default (with a DEBUG option), or enabled with a kernel option, or enabled with a /sys config.